app/components/dynamic-content.tsx

```
@@ -0,0 +1,35 @@
+    return (
+      <div className={className}>
+        <Component />
+      </div>
+    )
+  }
+
+  // If it's not a component name, treat it as HTML
+  return (
+    <div
+      className={className}
+      dangerouslySetInnerHTML={{ __html: content }}
```

Biome (1.9.4)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)